data.rb

```diff
@@ -0,0 +1,27 @@
+require /Users/kirillgavris/Desktop/Library/autoload.rb
+
+def save_library_data(file_name)
+    file_name = "Library.yml"
+    data = {
+      authors: YAML.dump(@authors),
+      books: YAML.dump(@books),
+      readers: YAML.dump(@readers),
+      orders: YAML.dump(@orders)
+    }
+    file = File.new(file_name, 'w')
+    file.write(YAML.dump data)
+  end
+
+  def load_library_data(file_name)
+    file_name = "load.yml"
+    if File.exist?(file_name)
+      file = File.open(file_name)
+      data = YAML.load file.read
+      @authors = YAML.load data[:authors]
+      @books = YAML.load data[:books]
+      @readers = YAML.load data[:readers]
+      @orders = YAML.load data[:orders]
+    else
+      puts "There is no such file."
+    end
+  end
```

Comment:
what is this? if you moved this logic into extentions/data.rb then delete this file